extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js

```diff
@@ -167,14 +167,17 @@ export class AmpStoryShoppingTag extends AMP.BaseElement {
     );
     const ratioOfLineHeightToFontSize = 1.5;
     const lineHeight = Math.floor(fontSize * ratioOfLineHeightToFontSize);
-    const height = textEl./*OK*/ clientHeight;
-    const numLines = Math.ceil(height / lineHeight);
 
-    this.mutateElement(() => {
-      pillEl.classList.toggle(
-        'i-amphtml-amp-story-shopping-tag-pill-multi-line',
-        numLines > 1
-      );
+    this.measureElement(() => {
+      const height = textEl./*OK*/ clientHeight;
+      const numLines = Math.ceil(height / lineHeight);
+
+      this.mutateElement(() => {
```

Comment:
Please get this `mutateElement` outside of the `measureElement`
To re-use `numLines`, you can do:

```
let numlines;

measureElement // use numLines
mutateElement // use numLines
```

Reply:
Yes, done.